### ISSUE TITLE
Is there a embed for broadcasting the video

### ISSUE THREAD

**Issue by krishdev** [ustream/embedapi#25]
Hi developers,

Here in the examples I can only see the player embed demos but I am looking something that I can embed on my webpage and broadcast the video rather going to ustream.tv or opening a desktop app.

**Comment by moczolaszlo** [comment]
Are you looking for a embedable brodcaster?

**Comment by krishdev** [comment]
Yes.. one that I can embed to the website and broadcast from there.
